Lib/_pyio.py

```diff
@@ -23,8 +23,8 @@
     valid_seek_flags.add(os.SEEK_HOLE)
     valid_seek_flags.add(os.SEEK_DATA)
 
-# open() uses st_blksize whenever we can
-DEFAULT_BUFFER_SIZE = 8 * 1024  # bytes
+# open() uses max(st_blksize, io.DEFAULT_BUFFER_SIZE) when st_blksize is available
+DEFAULT_BUFFER_SIZE = 128 * 1024  # bytes
```

Comment:
This value is used for the python only implementation of `io`. Performance considerations could be different for the python implementation, but I think it is good to indeed update this to the same value as in `_iomodule.h`.